docker/Dockerfile

```diff
@@ -128,7 +129,8 @@ RUN --mount=type=ssh \
     --mount=type=cache,id=workspace_cache,target=/workspace/.cache,sharing=locked \
     --mount=type=cache,id=conda_pkgs,target=/opt/conda/pkgs,sharing=locked \
     source activate base &&\
-    MORPHEUS_ROOT=/workspace CONDA_BLD_DIR=/opt/conda/conda-bld CONDA_ARGS="--no-test" ./ci/conda/recipes/run_conda_build.sh morpheus
+    # Temp add CONDA_CHANNEL_ALIAS to get around conda-build 404 errors
```

Comment:
Tech debt?

Reply:
Ya, its not great but hopefully this wont be a permanent thing. Worst case scenario is we use these mirrors for 22.04.